docs/js.html

```diff
@@ -1488,471 +1487,102 @@ <h1>Benchmark Report</h1>
-            [25, 7.291],
-            [50, 8.227],
-            [75, 9.125],
-            [90, 9.919],
-            [99, 13.019],
-            [99.99, 18.416],
-          ],
-        },
-        {
-          implementation: 'Sequelize',
-          duration: 10.01,
-          queries: 1662.68,
-          qps: 166.16,
-          latency_min: 4.784,
-          latency_mean: 6.009,
-          latency_max: 10.398,
-          latency_std: 0.463,
-          latency_cv: 7.7,
-          latency_percentiles: [
-            [25, 5.75],
-            [50, 5.975],
-            [75, 6.207],
-            [90, 6.452],
-            [99, 7.681],
-            [99.99, 10.405],
-          ],
-        },
-        {
-          implementation: 'Prisma',
-          duration: 10.01,
-          queries: 983.03,
-          qps: 98.23,
-          latency_min: 9.423,
-          latency_mean: 10.165,
-          latency_max: 12.935,
-          latency_std: 0.264,
-          latency_cv: 2.6,
-          latency_percentiles: [
-            [25, 10.001],
-            [50, 10.141],
-            [75, 10.285],
-            [90, 10.431],
-            [99, 11.151],
-            [99.99, 12.942],
-          ],
-        },
-        {
-          implementation: 'EdgeDB (Node.js, query builder)',
-          duration: 10.0,
-          queries: 4768.27,
-          qps: 476.74,
-          latency_min: 1.703,
-          latency_mean: 2.097,
-          latency_max: 8.341,
-          latency_std: 0.216,
-          latency_cv: 10.28,
-          latency_percentiles: [
-            [25, 2.016],
-            [50, 2.082],
-            [75, 2.147],
-            [90, 2.22],
-            [99, 2.553],
-            [99.99, 8.342],
-          ],
-        },
-      ];
+      var DATA_mean = [{"implementation": "TypeORM", "duration": 10.0, "queries": 1256.11, "qps": 125.61, "latency_min": 4.58, "latency_mean": 7.952, "latency_max": 17.379, "latency_std": 0.932, "latency_cv": 11.73, "latency_percentiles": [[25, 6.851], [50, 7.877], [75, 8.762], [90, 9.426], [99, 11.912], [99.99, 17.382]]}, {"implementation": "Sequelize", "duration": 10.01, "queries": 1328.12, "qps": 132.66, "latency_min": 4.896, "latency_mean": 7.523, "latency_max": 17.727, "latency_std": 0.806, "latency_cv": 10.7, "latency_percentiles": [[25, 6.936], [50, 7.776], [75, 8.045], [90, 8.281], [99, 9.451], [99.99, 17.738]]}, {"implementation": "Prisma", "duration": 10.01, "queries": 987.56, "qps": 98.69, "latency_min": 9.401, "latency_mean": 10.12, "latency_max": 13.438, "latency_std": 0.404, "latency_cv": 3.99, "latency_percentiles": [[25, 9.863], [50, 10.032], [75, 10.236], [90, 10.569], [99, 11.843], [99.99, 13.446]]}, {"implementation": "EdgeDB (Node.js, query builder)", "duration": 10.0, "queries": 4570.03, "qps": 456.9, "latency_min": 1.822, "latency_mean": 2.188, "latency_max": 7.596, "latency_std": 0.176, "latency_cv": 8.04, "latency_percentiles": [[25, 2.118], [50, 2.176], [75, 2.244], [90, 2.296], [99, 2.459], [99.99, 7.596]]}];
```

Comment:
It'd be nice to keep this pretty-printed.

Reply:
The file is generated, I've no idea why the old output was pretty and the new isn't.